fiftyone/utils/sam3.py

```
@@ -0,0 +1,610 @@
+            x_norm = max(0.0, x1 / width)
+            y_norm = max(0.0, y1 / height)
+            w_norm = (x2 - x1) / width
+            h_norm = (y2 - y1) / height
+            bounding_box = [
+                x_norm,
+                y_norm,
+                min(1.0 - x_norm, w_norm),
+                min(1.0 - y_norm, h_norm),
+            ]
+
+            y1_int = max(0, int(round(y1)))
```

Ruff (0.14.14)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)